lib/player.js

```diff
@@ -184,6 +184,21 @@ goog.requireType('shaka.media.PresentationTimeline');
  * @exportDoc
  */
 
+/**
+ * @event shaka.Player.AudioTrackChangedEvent
+ * @description Fired when the audio track changes at the playhead.
+ * That may be caused by an adaptation change or a DASH period transition.
+ * Separate events are emitted for audio and video contentTypes.
```

Comment:
It's audio track changed, so no emit for video.

Reply:
ah, yeah, I copied this from mediaqualitychanged at first.